libs/perun/dialogs/src/lib/mail-change-failed-dialog/mail-change-failed-dialog.component.html

```diff
@@ -0,0 +1,12 @@
+<h1 mat-dialog-title>{{'SHARED_LIB.PERUN.COMPONENTS.MAIL_CHANGE_FAILED_DIALOG.TITLE'| translate}}</h1>
+<div mat-dialog-content>
+  {{'SHARED_LIB.PERUN.COMPONENTS.MAIL_CHANGE_FAILED_DIALOG.INFO'|translate}}
```

Comment:
I'm just thinking if we could use here the `app-alert` component, which we have implemented for such notifications. This simple text doesn't look good and I think that if it would be with red color, the user would notice that it is a message for some error. What do you think?